AutomatedTesting/Gem/PythonCoverage/Code/Source/PythonCoverageEditorSystemComponent.h

```diff
@@ -74,11 +74,15 @@ namespace PythonCoverage
         void WriteCoverageFile();
 
         CoverageState m_coverageState = CoverageState::Disabled; //!< Current coverage state.
-        AZStd::unordered_map<AZStd::string, AZStd::unordered_map<AZ::Uuid, AZ::ComponentDescriptor*>> m_entityComponentMap; //!< Map of
-        //!< component IDs to component descriptors for all activated entities, organized by test cases.
+        AZStd::unordered_map<AZ::Uuid, AZ::ComponentDescriptor*> m_entityComponents; //!< Set of
```

Comment:
suggestion: Might be worth moving the comment to the line above with `//!` and avoid the slightly odd wrapping here?

Reply:
Fixed 👍